NEWS.md

```diff
@@ -41,6 +41,9 @@
 * in `describe` the specification of custom aggregation is now `function => name`;
   old `name => function` order is now deprecated
   ([#2401](https://github.com/JuliaData/DataFrames.jl/pull/2401))
+* `unstack` now does not depend on CategoricalArrays.jl and has two new keyword
```

Comment:
The dependency is a package-level property, for users it doesn't matter whether the implementation of `unstack` depends on CategoricalArrays. This formulation could be interpreted as meaning that the behavior changed.

Reply:
I will update the description when we settle if we want to sort or not